octavia_proxy/tests/functional/api/v2/base.py

```diff
@@ -150,7 +168,7 @@ def _get_sdk_connection(self):
 
     def _create_network(self):
         global _network
-        cidr = '192.168.0.0/16'
+        cidr = CIDR
```

Comment:
why did you change the subnet to 'CIDR' ?

Reply:
fixed